custom_components/amplifi/device_tracker.py

```diff
@@ -145,6 +145,14 @@ def extra_state_attributes(self):
             return {**self._data, "last_seen": datetime.now().isoformat()}
         return {}
 
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if the entity should be enabled when first added to the entity registry."""
+        if self.config_entry.data[CONF_ENABLE_NEW_DEVICES]:

```

Comment:
As part of testing an upgrade of an existing install in my test instance, this threw a KeyError.
I suggest maybe doing a get() to handle cases where the new config item doesn't exist yet:

`if self.config_entry.data.get(CONF_ENABLE_NEW_DEVICES, False):`